CSharpBashInterpreter/Commands/Abstractions/BaseCommandRepresentation.cs

```diff
@@ -6,5 +6,9 @@ public abstract class BaseCommandRepresentation : ICommandRepresentation
 {
     public abstract string Name { get; }
     public abstract ICommandExecutable Build(IEnumerable<string> tokens);
-    public virtual bool CanBeParsed(IEnumerable<string> tokens) => tokens.First() == Name;
+
+    public virtual bool CanBeParsed(IEnumerable<string> tokens)
+    {
+        return tokens.First() == Name;
+    }
```

Comment:
```suggestion
    public virtual bool CanBeParsed(IEnumerable<string> tokens)
        => tokens.First() == Name;
```
Это вроде читаемости не портит.

Reply:
Fixed